test_tasks/task5.py

```diff
@@ -0,0 +1,15 @@
+def fun2(value):
+    value = str(value)
+
+    if value == value[::-1]:
+        return True
+    else:
+        return False
```

Comment:
`input` returns `str`
```suggestion
    if value == value[::-1]:
        return True
    return False
```